modules/clients/src/test/java/org/apache/ignite/internal/jdbc2/JdbcMetadataSelfTest.java

```diff
@@ -320,10 +319,8 @@ public void testGetTableTypes() throws Exception {
      */
     @Test
     public void testGetAllView() throws Exception {
-        List<String> expViews = Arrays.asList(
+        Set<String> expViews = new HashSet<>(Arrays.asList(
             "BASELINE_NODES",
-            "CACHES",
-            "CACHE_GROUPS",
```

Comment:
Wouldn't it be better to have enabled system view by default? To enable views admin must restart the node, sometimes it's not a good issue.

Reply:
I see your point.
Let's discuss it on the dev-list?

Seems, it's up to community consensus.